src/mcmc/Inference.jl

```diff
@@ -1,9 +1,13 @@
 module Inference
 
 using DynamicPPL:
+    DynamicPPL,
     @model,
     Metadata,
     VarInfo,
+    LogDensityFunction,
+    SimpleVarInfo,
+    AbstractVarInfo,
```

Comment:
Being pedantic and annoying, I think it'll be better if we reorder so that the `*VarInfo`s stay together.